packages/tokens/properties/color/font.json

```diff
@@ -14,10 +14,10 @@
         "value": "{color.grey.900.value}"
       },
       "dark": {
-        "value": "{color.grey.800.value}"
+        "value": "{color.grey.700.value}"
       },
       "light": {
-        "value": "{color.grey.700.value}"
+        "value": "{color.grey.500.value}"
       },
       "lightest": {
         "value": "{color.grey.000.value}"
```

Comment:
Sorry @tiloyi, after checking, we have to use this values for a better accessibility:

"dark": {
        "value": "{color.grey.800.value}"
},
"light": {
        "value": "{color.grey.600.value}"
},

Reply:
fixed